doc/source/building/working_with_images/custom_volumes.rst

```diff
@@ -0,0 +1,64 @@
+.. _custom_volumes:
+
+Custom Disk Volumes 
+===================
+
+.. sidebar:: Abstract
+
+   This page provides high level explanations on how to handle volumes or
+   subvolumes definitions for disk images using KIWI.
+
+KIWI supports defining custom volumes by using the logical volume manager
+for the Linux kernel (LVM) or by setting volumes at filesystem level when
+filesystem supports it, like Btrfs.
+
+Volumes are defined into the KIWI description file, :file:`config.xml`,
+under the `type` element by setting a child `systemdisk` section. The
+`systemdisk` element can include child `volume` elements. 
+
+Additional non root volumes are created for each `volume` entry. Volume
```

Comment:
1. non root -> non-root
1. entry? Is it an element? I would stick with element.